flux/combineFiles_test.flux

```diff
@@ -1,9 +1,9 @@
 default folderPath="test/output";
 default outfile="test/output/combinedDe-Sol1Holdings.tsv";
-
+default fileName="sol1Holding_(seq|sru)\\.tsv";
 
 folderPath
-| read-dir(filenamepattern="sol1Holding_.*\\.tsv")
+| read-dir(filenamepattern=fileName)
```

Comment:
I think using camel case in the options makes them more readable, i.e. `fileNamePattern` (not just here but in general).

Reply:
Should be fixed by: https://github.com/hbz/lobid-extra-holdings/pull/14/commits/367112c36b99252220b6b6e96ce795616911beee